code/datums/mutations/body.dm

```diff
@@ -173,6 +173,9 @@
 	locked = TRUE //Species specific, keep out of actual gene pool
 
 /datum/mutation/human/race/on_acquiring(mob/living/carbon/human/owner)
+	if(owner.has_brain_worms())
```

Comment:
well that's what the borer had, and since it can also affect any other """"future"""" brain worms, i figured it's as fine as it is, if it affects them all